fastlane/lib/fastlane/helper/adb_helper.rb

```diff
@@ -30,7 +30,7 @@ def trigger(command: nil, serial: nil)
         Action.sh(command)
       end
 
-      def device_avalaible?(serial)
+      def device_available?(serial)
```

Comment:
Couldn't find any usage in `fastlane`'s code base, however I can imagine some might use this helper method in a `Fastfile`. 

```
lane :custom_lane do
      adb = Helper::AdbHelper.new(adb_path: path)
      puts "Device is available" if adb.device_avalaible?(serial)
end
```

[The documentation](https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/helper/README.md) describes how to do that. 

I guess the question is if this change is a breaking change? 🤔

Reply:
@mollyIV I searched all over the [GitHub](https://github.com/search?p=5&q=device_avalaible%3F&type=Code), I couldn't find any custom usage of `device_avalaible?`, I think it might be safe, but not sure :smiling_imp:

Reply:
This is a great catch but also _technically_ a breaking change here 🙃 I would probably recommend doing something like this...

```rb
def device_avalaible?(serial)
  UI.deprecated("Please use `device_available?` instead... This will be removed in a future version of fastlane")
  device_available?(serial)
end

def device_available?(serial)
```

Reply:
Going to make this change so that we can get this merged ^